scalafmt-tests/src/test/resources/scala3/Macro.stat

```diff
@@ -45,7 +45,37 @@ val xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx = ${    locationImpl(Foooooooooooooooooooooo
 val xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx = ${
   locationImpl(Foooooooooooooooooooooooo())
 }
-<<< SKIP dollar parameter - FIXME https://github.com/scalameta/scalameta/issues/2191
+<<< dollar parameter
 List().map(($, a) => $.a)
 >>>
 List().map(($, a) => $.a)
+<<< simple pattern
+   body match    {  
+      case '{ sum() } => Expr(0)
+      case '{ sum($n) } =>        n
+      case '{ sum(${Varargs(args)}: _*) } => sumExpr(args)
+      case body => body
+   }
+>>>
+body match {
+  case '{ sum() }                       => Expr(0)
+  case '{ sum($n) }                     => n
+  case '{ sum(${ Varargs(args) }: _*) } => sumExpr(args)
+  case body                             => body
+}
+<<< long pattern
```

Comment:
should we add tests with `newlines.source=fold/etc.`?

Reply:
Added just in case, but everything seems to work fine.